scripts/mk_project.py

```diff
@@ -9,7 +9,7 @@
 
 # Z3 Project definition
 def init_project_def():
-    set_version(4, 5, 1, 0)
+    set_version(4, 6, 1, 0)
```

Comment:
@NikolajBjorner 
Unfortunately you need to update the version in the root `CMakeLists.txt` too.

https://github.com/Z3Prover/z3/blob/d7042c234f32607c01324da7a3bbf74384d0aaab/CMakeLists.txt#L32-L41

I never refactored this because the intention was for the CMake build system to replace the older one.

Reply:
ok